db/seeds.rb

```diff
@@ -0,0 +1,33 @@
+require "bundler/setup"
+
+require "pg"
+require "active_record"
+require "pry"
+
+require_relative "connection"
+require_relative "../models/apartment"
+require_relative"../models/tenant"
+
+Apartment.create address ([
+  {address: "123 Main Street", monthy_rent: 1900, sqft: 650 num_beds: 1, num_baths: 1},
+  {address: "123 Steve Street", monthy_rent: 2200, sqft: 950 num_beds: 2, num_baths: 1.5},
+  {address: "256 Jason Street" monthly_rent: 3500, sqft: 1200, numb_beds: 3, num_baths: 2.5}
+  ])
```

Comment:
That `address` after `Apartment.create` is going to cause your code to error out. It should just look like...

```
Apartment.create([
  ...
)]
```